aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -538,8 +540,8 @@ Tensor special_sinc(const Tensor& self) { return self.sinc(); }
 namespace {
 
 inline Tensor calc_ndtr(const Tensor& self) {
-  auto x_sqrt_2 = self / std::sqrt(2.);
-  return (1 + at::erf(x_sqrt_2)) * 0.5;
+  auto x_sqrt_2 = self * M_SQRT1_2;
+  return at::erfc(-x_sqrt_2) * 0.5;
```

Comment:
is erf(-x) == -erf(x)?

Reply:
yes... 
erf(-x) == -erf(x)
erfc(-x) == 2 - erfc(x)
erfc(x) == 1 - erf(x)